_includes/payment-order-other-features.md

```diff
@@ -433,10 +433,10 @@ The `view-paymentorder` operation contains the URI of the JavaScript that needs
                 },
                 onPaymentToS: function(paymentToSEvent) {
                     console.log(paymentToSEvent);
-                },
+                },{% if documentation_section == "payment-menu" %}
```

Comment:
I'm not 100% sure `onPaymentMenuInstrumentSelected` should be hidden from Checkout. Can you please investigate?

Reply:
Did you miss this?